From b85756ea5a3a19d2ad227d71490ee3ba6cda0e02 Mon Sep 17 00:00:00 2001
From: Yijie Yang <quic_yijiyang@quicinc.com>
Date: Mon, 11 Nov 2024 14:07:53 +0800
Subject: [PATCH 40/68] dt-bindings: net: qcom,ethqos: revise description for
 qcs615

The core version of EMAC on qcs615 has minor differences compared to that
on sm8150. During the bring-up routine, the loopback bit needs to be set,
and the Power-On Reset (POR) status of the registers isn't entirely
consistent with sm8150 either.
Therefore, it should be treated as a separate entity rather than a
fallback option.

Fixes: 32535b9410b8 ("dt-bindings: net: qcom,ethqos: add description for qcs615")
Signed-off-by: Yijie Yang <quic_yijiyang@quicinc.com>
---
 Documentation/devicetree/bindings/net/qcom,ethqos.yaml | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Documentation/devicetree/bindings/net/qcom,ethqos.yaml b/Documentation/devicetree/bindings/net/qcom,ethqos.yaml
index 0bcd593a7bd0..576a52742ff4 100644
--- a/Documentation/devicetree/bindings/net/qcom,ethqos.yaml
+++ b/Documentation/devicetree/bindings/net/qcom,ethqos.yaml
@@ -23,12 +23,9 @@ properties:
           - enum:
               - qcom,qcs8300-ethqos
           - const: qcom,sa8775p-ethqos
-      - items:
-          - enum:
-              - qcom,qcs615-ethqos
-          - const: qcom,sm8150-ethqos
       - enum:
           - qcom,qcs404-ethqos
+          - qcom,qcs615-ethqos
           - qcom,sa8775p-ethqos
           - qcom,sc8280xp-ethqos
           - qcom,sm8150-ethqos
-- 
2.34.1

